eufy_security/p2p/discovery.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+
```

Comment:
I think we need to import logging here and add something like:
`_LOGGER: logging.Logger = logging.getLogger(__name__)`